### ISSUE TITLE
TimeGrid : Not an issue, I ask for better understanding

### ISSUE THREAD

**Issue by chaklasiyanikunj** [rokudev/SceneGraphDeveloperExtensions#19]
I check TimeGridView Example. Create a log and try to understand. Anyone know how many days of data available in this example. and I don't understand one thing every time launch application. It means I launch at 4 PM then the data display starts at 3 PM. I lunch at 5 PM then the data display starts at 4 PM.

**Comment by RokuChris** [comment]
The times in the sample channel are contrived so that it will always show data for the current period. This happens in the `CHRow` ContentHandler. In a real world channel you would replace this with logic to get the _playStart_ values from your API.

```
    ' we'll fake the timestamps for demo purposes so that there Is
    ' always content displayed for the current time
    dt = CreateObject("roDateTime")
    now = dt.AsSeconds()
    playStart = now - (now mod 1800) - 3600
```

**Comment by chaklasiyanikunj** [comment]
Thank you for your Great Answer.

**Comment by chaklasiyanikunj** [comment]
It's Possible to change Program FontSize. I analyze the 2nd API Font in Label. But How to set for 3rd API title Font.

**Comment by chaklasiyanikunj** [comment]
I find some observations regarding the timegridview.
1. Sometimes the focus on program name is not maintained. like below. here below two screenshots are available. 
              a. this is the program displaying the starting point. (program duration: 38 Min)
![1-a](https://user-images.githubusercontent.com/39697799/87851832-03e6a000-c91a-11ea-8226-d3725ec2b22a.png)
              b. I continued to press the right key and immediately stop, then I face the issue. (program duration: 66 Min)
![1-b](https://user-images.githubusercontent.com/39697799/87851841-1d87e780-c91a-11ea-92c3-2b686f48cfb8.png)

2. Sometimes I show content above the content in a program name like below screenshot in the current displaying program name. (First program duration: 46 Min Second program duration: 58 Min)
![2](https://user-images.githubusercontent.com/39697799/87851845-30022100-c91a-11ea-90f3-5fdca118c93e.png)

3. Sometimes My duration is small and the program name is big. It shows the below screenshot. 
     (1.) "..." is the 6 min duration program.
     (2.) the black rectangle is 4 min duration program. 
![3](https://user-images.githubusercontent.com/39697799/87851863-5627c100-c91a-11ea-91bf-3bf99db81f17.png)

this all things are happening because the current version is required 3o min or 1-hour program. right? Is it solved that points in the next timegridview version?

Like a 3rd Point, I created a custom time grid. I added ScrollingLabel in a custom time grid For displaying a Full Program name. when the focus is available to program name is scrolling But still, it's not visible proper. Is it possible to add a tooltip in Roku? or any suggestions which should I use for displaying a full program.
